--- a/out/prod/extension-prod.js
+++ b/out/prod/extension-prod.js
@@ -129255,7 +129255,6 @@
         activationEvent.addProperty(telemetryStrings_1.TelemetryPropertyNames.FIRST_LIVESHARE_ACTIVATION, true);
         await welcomePageUtil_1.showWelcomePage(welcomePageUtil_1.WelcomePageSource.Install);
     }
-    await extensionUtil_1.ExtensionUtil.updateExecutablePermissionsAsync();
     await launcher_1.Launcher.setup(false, !(await downloader_1.installFileExistsAsync()));
     activationEvent.markTime(telemetryStrings_1.TelemetryPropertyNames.EXTENSION_ACTIVATION_LAUNCHER_SETUP_COMPLETE);
     setupRpcFilters(rpcClientFactory_1.rpcClientFactory());